fe/fe-core/src/main/java/com/starrocks/metric/MaterializedViewMetricsRegistry.java

```diff
@@ -67,44 +63,40 @@ public void run() {
 
     // collect materialized-view-level metrics
     public static void collectMaterializedViewMetrics(MetricVisitor visitor, boolean minifyMetrics) {
-        GlobalStateMgr globalStateMgr = GlobalStateMgr.getCurrentState();
-        List<String> dbNames = globalStateMgr.getLocalMetastore().listDbNames();
-        for (String dbName : dbNames) {
-            Database db = GlobalStateMgr.getCurrentState().getDb(dbName);
-            if (null == db) {
+        MaterializedViewMetricsRegistry instance = MaterializedViewMetricsRegistry.getInstance();
```

Comment:
Is there some stale entries in this registry need to clean?

Reply:
Yep. there are a background cleaner to do this every day.

Reply:
MaterializedViewMetricsRegistry#MetricsCleaner